gtest/src/program.rs

```diff
@@ -351,7 +351,7 @@ impl<'a> Program<'a> {
         }
 
         let program_id = id.clone().into().0;
-        let program = CoreProgram::new(program_id, code);
+        let program = CoreProgram::new(program_id, Default::default(), code);
```

Comment:
I think nonce should be used or 2 programs on 1 system are impossible

Reply:
Two different programs with an equal id?